projects/archiver/src/notifications/device-notifications.ts

```diff
@@ -1,30 +1,59 @@
-import { IssuePublicationIdentifier, Issue } from '../../common'
+import uuid from 'uuidv4'
+import { attempt } from '../../../backend/utils/try'
+import fetch from 'node-fetch'
+import dateFormat from 'dateformat'
 
-interface Topic {
-    type: string
+type EditionNotificationTypes = 'editions'
+
+type NotificationSenders = 'editions-backend'
+
+interface EditionTopic {
+    type: EditionNotificationTypes
     name: string
 }
 
 interface ScheduleAppStoreNotificationPayload {
     id: string
-    type: string
-    topic: Topic[]
+    type: EditionNotificationTypes
+    topic: EditionTopic[]
     key: string
     name: string
     date: string
-    sender: string
+    sender: NotificationSenders
+}
+
+const createDatePathParam = (now: Date) => {
+    const THREE_AT_NIGHT = 'T03:00:00Z'
+    return `${dateFormat(now, 'yyyy-mm-dd')}${THREE_AT_NIGHT}`
+}
```

Comment:
This is reasonable for now but we'll need to revisit when we make this work generically for different editions so can you add a TODO to that effect?

Reply:
done